x-pack/plugins/ml/public/application/components/custom_urls/custom_urls.tsx

```diff
@@ -69,14 +87,16 @@ class CustomUrlsUI extends Component<CustomUrlsProps, CustomUrlsState> {
       dataViewListItems: [],
       queryEntityFieldNames: [],
       editorOpen: false,
+      supportedFilterFields: [],
     };
   }
 
   static getDerivedStateFromProps(props: CustomUrlsProps) {
     return {
       job: props.job,
       customUrls: props.jobCustomUrls,
-      queryEntityFieldNames: getQueryEntityFieldNames(props.job),
+      // For DFA uses the destination index Data View to get the query entities and falls back to source index Data View.
+      queryEntityFieldNames: getDropDownOptions(props.job, props.dataView),
```

Comment:
I don't think we need the `ml.feature_importance` fields available here. I'm struggling to think where they would be useful in the target page.

<img width="357" alt="image" src="https://user-images.githubusercontent.com/7405507/230398284-ab80206e-07bc-407f-97bd-1dace1f47731.png">

Reply:
Removed results fields in 5b70da1b2fa75a6287841fa875d87c64387d1af6